kernel/src/arch/x86_64/network.rs

```diff
@@ -0,0 +1,73 @@
+use alloc::collections::BTreeMap;
```

Comment:
should this file be in x86_64/exokernel?

Reply:
Ah, I see your point. I was thinking since we only use it in exokernel maybe move it in there for now, if we expand it to other archs later, we can move it in the base?